dev_guide/deployments.adoc

```diff
@@ -190,32 +190,28 @@ responsibility of the strategy is to make the new deployment live using the
 logic that best serves the needs of the user.
 
 === Deployment resources
-A deployment is completed by a pod that consumes resources on a node.
+A deployment is completed by a pod that consumes *resources* (memory and cpu) on a node.
+Normally, pods consume until project-default limits are reached, or use unbounded node resources if no such limits are specified.
```

Comment:
By default, a deployment pod uses unbounded node resources.  If a project has specified default container resource limits, the deployment pod's containers will consume that amount of resource.